styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+  margin: 0;
+  padding: 0;
+  background-color: #191765;
+  font-family: -apple-system, Raleway, Roboto, Oxygen, sans-serif;
+}
+
+.toolbar {
+  min-height: 4rem;
+  display: flex;
+  align-items: center;
+  padding: 15px;
+  color: white;
+  opacity: 1;
+}
+
+.toolbar-items {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+}
+
+.bar-icon {
+  zoom: 1.5;
+}
+
+.bar-icon:hover {
+  zoom: 1.8;
+  transition: ease-out;
+  box-shadow: 1px 1px 5px gray;
+}
+
+#logo-text {
+  line-height: 2px;
+  font-weight: 500;
+}
+
+.headline-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  background: url("/headline_bg.svg");
```

Comment:
- [ ] I believe that there is a pathing issue, when I open your portfolio it does not show up the background image. You should consider the following: `background: url("./headline_bg.svg");`.

> Keep in mind that what shows up in your IDE live server is not necessarily what's going to be displayed when your website is deployed. IDE's are "smarter", so small pathing issues like this one are automatically fixed!

Figma design:
![image](https://user-images.githubusercontent.com/66967274/148391388-1a8a16d7-642a-472d-8d47-55ce437a4058.png)


Your project:
![image](https://user-images.githubusercontent.com/66967274/148391279-48f8840e-3af4-4e53-89d0-7764102aaf3c.png)